### PR TITLE
Add global output directory and rename test output directory

### LINKED ISSUE #181: Global syntest output directory config
Currently, the plugins have to define their own plaths where to store data. This could recreate some inconsistencies and would also require many changes to update the directory.

I propose we make a global config parameter for the main output folder.

_Originally posted by @mitchellolsthoorn in https://github.com/syntest-framework/syntest-core/pull/180#discussion_r1091883855_

### PR DESCRIPTION
**Pull request by dstallenberg**
Closes #181

### CODE REVIEW

**Comment by dstallenberg** [comment]
I only added  the main folder property. (/syntest)
The statistics folder is still specified as a full path (/syntest/statistics).
This allows the user to change the output path of for example only the statistics output (and not the test output)

File: packages/core/lib/Configuration.ts [hunk 1]
```diff
@@ -157,6 +157,15 @@ export class Configuration {
   configureStorageOptions<T>(yargs: Yargs.Argv<T>) {
     return yargs.options({
       // directories
+      "syntest-directory": {
```
Comment (mitchellolsthoorn) [inline packages/core/lib/Configuration.ts:160]: What is saved in this directory at the moment, If we keep both the final test cases and the statistics separate?

Comment (dstallenberg) [reply packages/core/lib/Configuration.ts:160]: Nothing, it just specifies the main folder.

Comment (mitchellolsthoorn) [reply packages/core/lib/Configuration.ts:160]: However, if the tests and the statistics are stored in their own separate path, what is the use case for this  `syntest-directory`?

Comment (dstallenberg) [reply packages/core/lib/Configuration.ts:160]: It's mostly meant for plugin to be able to use the main folders.
I think its still valuable to be able to pick other locations for the statistics and test cases.

**Review by mitchellolsthoorn — APPROVED** [review]

**Pull request merged**